README.md

```diff
@@ -28,19 +25,18 @@
 
 <a href="https://xn--931a.moe/"><img src="https://github.com/misskey-dev/misskey/blob/develop/assets/ai.png?raw=true" align="right" height="320px"/></a>
 
-## ✨ Features
+## ✨ Highlights
 - **ActivityPub support**\
-	It is possible to interact with other software.
+	Federates well with Mastodon, Pleroma and more.
 - **Reactions**\
-	You can add "reactions" to each post, making it easy for you to express your feelings.
+	Choose from a palette of reactions to express your feelings.
```

Comment:
You can add emoji reactions to posts! Choose from default or custom emojis to share your feelings.

Reply:
Why lengthen it excessively?